docs/user-guide/plotting.ipynb

```diff
@@ -24,10 +24,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "There are currently three different backends for plotting. The default (`'interactive'`) renders interactive plots for Jupyter notebooks, while the second (`'static'`) generates static png exports of the figures.\n",
-    "The third (`'matplotlib'`) returns a dict of `matplotlib` objects that can then be used in highly customized figures. There is an additional `'matplotlib:quiet'` backend which does not return the objects dict, but is used to simply render the basic `matplotlib` figures in the Jupyter notebook.\n",
-    "\n",
-    "Here we switch to the `matplotlib` backend, as `plotly` (interactive and static) figures currently do not work when embedded in the documentation pages on `Read the Docs`."
+    "It is possible to render either static or interactive graphs. By default, the static (`inline`) backend is used. To activate the interactive (`notebook`) backend, simply run the `%matplotlib notebook` magic command (uncomment the cell below)."
```

Comment:
This default makes sense for our documentation, but I suppose almost everyone using this in Jupyter would like the interactive one as default? Or is there a drawback?